packs/zookeeper/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
+zookeeper_hosts: "localhost:2181"
+zookeeper_root: "/"
```

Comment:
We could make it overridable in action params, but this was the simplest thing that solved the immediate need. My basic reasoning is that StackStorm users don't (usually) care about URLs, so specify them system-wide. Of course, that assumption fails when you have multiple Zookeeper clusters that you want to integrate with; but we can cross that bridge when we come to it. :)